devspaces/instance/NOTES.md

```diff
@@ -0,0 +1,34 @@
+# Notes
```

Comment:
Any reason for making this a notes.md file instead of a readme?

I don't think that notes will show up in the github UI by default when you are browsing the folder.

Reply:
These are general notes about Dev Spaces. Didn't mean to put them with the instance. They are outside of a regular readme.